modules/deploy/pages/deployment-option/cloud/configure-privatelink-in-cloud-ui.adoc

```diff
@@ -0,0 +1,49 @@
+
+NOTE: This guide is for configuring AWS PrivateLink using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] if you want to set up the endpoint service using the API.
+
+The Redpanda AWS PrivateLink endpoint service provides secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink does not go through the public internet because these connections are treated as their own private AWS service. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using the endpoint service if you have multiple VPCs and could benefit from a more simplified approach to network management:
+
+* PrivateLink allows overlapping xref:./cidr-ranges.adoc[CIDR ranges] in VPC networks.
+* PrivateLink does not limit the number of connections that use the endpoint service.
+* You control which AWS principals are allowed to connect to the endpoint service.
+
+== Requirements
+
+* Your Redpanda cluster and VPC must be in the same region.
+* Use the https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html[AWS CLI] to create a new client VPC or modify an existing one to use the PrivateLink endpoint.
+
+== Enable endpoint service for existing clusters
+
+. In the Redpanda Cloud UI, open your https://cloud.redpanda.com/clusters[cluster^], and click **Cluster settings**.
+. Under AWS PrivateLink, click **Enable**. 
+. You need the Amazon Resource Names (ARNs) for the AWS principals allowed to access the endpoint service. For example, for all principals in a specific account, use `arn:aws:iam::<account-id>:root`. See the official AWS documentation on https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#add-remove-permission[configuring an endpoint service^] for details.
+. It may take several minutes for your cluster to update. When the update is complete, the AWS PrivateLink status in **Cluster settings** changes from **In progress** to **Enabled**.
+
+NOTE: For help with issues with enabling PrivateLink, contact https://support.redpanda.com/hc/en-us/requests/new[Redpanda support^].
+
+== Access Redpanda services through VPC endpoint
+
+After you have enabled PrivateLink for your cluster, your connection URLs are available in the *How to Connect* section of the cluster overview in the Redpanda Cloud UI.
+
+include::deploy:partial$cloud/private-links-access-rp-services-through-vpc.adoc[]
+
+== Test the connection
+
+You can test the connection to the endpoint service from any VM or container in the consumer VPC. If configuring a client isn't possible right away, you can do these checks using `rpk` or cURL:
+
+include::deploy:partial$cloud/private-links-test-connection.adoc[]
+
+== Disable endpoint service
+
+In **Cluster settings**, click **Disable**. Existing connections are closed immediately. To connect via PrivateLink again, you'll have to reenable the service.
```

Comment:
Suggest to change `Existing connections are closed immediately` to `Existing connections will be closed after AWS PrivateLink service is disabled.`